compose/config/config.py

```diff
@@ -493,19 +490,14 @@ def process_config_file(config_file, environment, service_name=None):
             config_file.get_networks(),
             'network',
             environment)
-        if config_file.version in (V3_1,):
+        if config_file.version in (const.COMPOSEFILE_V3_1, const.COMPOSEFILE_V3_2):
```

Comment:
Specifically in that case, it messes up with a validation step later on because there's an unexpected `secrets` key in the resulting dictionary